src/driftpy/accounts/oracle.py

```diff
@@ -73,9 +57,26 @@ async def _parse_pyth_price_info(
 
     offset += 160
 
-    return (
-        PythPriceInfo.deserialise(buffer, offset, exponent=exponent),
-        last_slot,
-        twac,
-        twap,
+    pyth_price_info = PythPriceInfo.deserialise(buffer, offset, exponent=exponent)
+
+    scale = 1
+    if "1K" in str(oracle_source):
+        scale = 1e3
+    elif "1M" in str(oracle_source):
+        scale = 1e6
```

Comment:
todo: add support for the stable type

Reply:
yeah and there is an issue with Pyth versus PYTH